src/templates/faq-page.js

```diff
@@ -16,8 +16,10 @@ export const FaqPageTemplate = ({
   ajaxRequestStates = {},
   plusIcon = {},
 }) => {
+  const [expandedId, setExpandedId] = useState(-1)
+
   return (
-    <div className="main-content faq">
+    <div className="faq-content">
       <PageSection id={sections.faq.HOME}>
         <Row>
           <Col xs={12}>
```

Comment:
@maymax777, You missed the title's underline (green-rect). You can make it as following:
```
<h1 className="h1-underline">{hero.title}</h1>
```
So, we don't need `green-rect.svg` file actually.